openssl/src/symm.rs

```diff
@@ -534,4 +588,42 @@ mod tests {
 
         cipher_test(super::Cipher::des_ecb(), pt, ct, key, iv);
     }
+
+    #[test]
+    fn test_aes128_gcm() {
+        let key = "0e00c76561d2bd9b40c3c15427e2b08f";
```

Comment:
The hex strings are copied directly out of the NIST test vectors, where they're provided hex encoded. I could stick the decode right on the string literals I guess.